lib/ElementHandle.js

```diff
@@ -103,6 +103,26 @@ class ElementHandle {
     const objectId = this._remoteObject.objectId;
     return this._client.send('DOM.setFileInputFiles', { objectId, files });
   }
+
+  /**
+   * @return {!Promise<Object>}
+   */
+  async boundingBox() {
+    const box = await this.evaluate(element => {
+      if (!element.ownerDocument.contains(element))
+        return null;
+      const rect = element.getBoundingClientRect();
```

Comment:
I've added the test above, but I ended up using the code from [document-offset](https://github.com/timoxley/offset) for getting the absolute offset.